api/src/pcapi/routes/backoffice/templates/individual_bookings/list.html

```diff
@@ -198,6 +199,9 @@ <h5 class="modal-title">Annuler la réservation {{ booking.token }}</h5>
                 <td>{{ booking.stock.quantity }}</td>
                 <td>{{ booking.total_amount | format_amount }}</td>
                 <td>{{ booking | format_booking_status(with_badge=True) | safe }}</td>
+                <td>
+                  {% if booking.status.name == "USED" %}{{ booking.validationAuthorType | format_booking_validation_author_type }}{% endif %}
```

Comment:
Je ne vois pas pourquoi on ne voudrait pas avoir cette information affichée dans le cas d'une réservation remboursée (`booking.status.name == "REIMBURSED"`), qui est la suite logique de l'état `USED`. Il n'est donc pas souhaitable de merger en l'état.

À moins de vouloir explicitement ne pas afficher cette colonne si la réservation a finalement été annulée après avoir été validée (le ticket ne le précise pas), on doit pouvoir tout simplement écrire : 
```suggestion
                  {{ booking.validationAuthorType | format_booking_validation_author_type }}
```